script.js

```diff
@@ -0,0 +1,10 @@
+$(document).ready(function() {
+    alert('Hello,world!');
+});
+
+
+$('#test').html;
```

Comment:
It looks like you are trying to set the html of your test div - but you aren't doing this in a function and you don't set this to a value.